src/main/java/UI.java

```diff
@@ -2,9 +2,14 @@
  * Encapsulates messages for interacting with the user.
  */
 
+/**
+ * manages user interfacce interactions
```

Comment:
I think you forgot to capitalize the first letter and made a typo for 'interface'